DSharpPlus/Entities/DiscordMentions.cs

```diff
@@ -33,21 +33,26 @@ internal class DiscordMentions
         /// </summary>
         [JsonProperty("parse", NullValueHandling = NullValueHandling.Ignore)]
         public IEnumerable<string> Parse { get; }
+        
+        [JsonProperty("replied_user", NullValueHandling = NullValueHandling.Ignore)]
+        public bool? RepliedUser { get; }
 
-        internal DiscordMentions(IEnumerable<IMention> mentions)
+        internal DiscordMentions(IEnumerable<IMention> mentions, bool mention = false)
```

Comment:
What is the purpose of this?

Reply:
I vaguely remember glancing over the API docs, and the mention user is a boolean, but if you have some better way to handle this, I'd be more than happy to change it

Reply:
This is the way it is handled on the [Allowed Mentions](https://discord.com/developers/docs/resources/channel#allowed-mentions-object-allowed-mentions-structure) object structure defined in the docs.

Reply:
Ah my bad, I didn't see it being assigned to the `RepliedUser`